.github/workflows/deploy.yml

```diff
@@ -85,15 +85,16 @@ jobs:
 
       - name: Terraform init
         run: |
-          terraform init -backend-config=terraform/workspace-variables/${{ env.DEPLOY_ENV }}_backend.tfvars terraform
+          cd terraform
+          terraform init -backend-config=workspace-variables/${{ env.DEPLOY_ENV }}_backend.tfvars
         env:
           ARM_ACCESS_KEY: ${{ secrets[format('ARM_ACCESS_KEY_{0}', env.DEPLOY_ENV)] }}
           TF_VAR_paas_app_docker_image: ${{env.DOCKER_REPO}}:${{ github.event.inputs.sha }}
 
       - name: Terraform Plan and Apply
         run: |
-          terraform plan -var-file=terraform/workspace-variables/${{ env.DEPLOY_ENV }}.tfvars terraform
-          terraform apply -var-file=terraform/workspace-variables/${{ env.DEPLOY_ENV }}.tfvars -auto-approve terraform
+          terraform plan -var-file=workspace-variables/${{ env.DEPLOY_ENV }}.tfvars
```

Comment:
not sure `cd terraform` in the previous task will persist in this action,
you need to add `working-directory: terraform` here too

Reply:
done...